### ISSUE TITLE
Unreadable date

### ISSUE THREAD

**Issue by bckthomas** [Kibibit/hass-kibibit-theme#14]
Hi
Since the last update, the dates are unreadable with the calendar card.
Can you do something ?
Thanks
Kibibit 1.1.1
Home assitant 0.118.5
![Screenshot_2020-12-07 Home Assistant](https://user-images.githubusercontent.com/57676204/101398815-f3d25b80-38ce-11eb-9e78-02b76b62fe23.png)

**Comment by Thatkookooguy** [comment]
you're taking about the white bars? wasn't sure if you added that to obfuscate the data or is that the card itself :-)

I'll check it out!

**Comment by F1DNA** [comment]
I went through and replaced everything with '#0984ff' to locate where this is coming from.  I am fairly certain I tried every variable in your yaml.  Either this is coming from somewhere else or I got unlucky and frontend.reload_themes failed me when I hit the right variable.

**Comment by bckthomas** [comment]
If this can help, the color is defined here : 

![image](https://user-images.githubusercontent.com/57676204/103074509-0684aa00-45ca-11eb-9172-470eefb9de7f.png)

**Comment by Thatkookooguy** [comment]
@bckthomas it does! Thanks!

**Comment by thatkookooguy** [comment]
@bckthomas  @F1DNA does this still happen for you? tried to look at this today and everything looked fine

![image](https://user-images.githubusercontent.com/10427304/106328981-5e799500-6289-11eb-9202-1d17ab41a5ae.png)

**Comment by F1DNA** [comment]
Shortly after posting my comment, I tried out the Atomic Calendar Revive card and prefer that over the stock one.  This issue wasn't present on that card so it was a win/win.  I could load up the stock card soon and report back, if you would like.

**Comment by F1DNA** [comment]
All of the views with the stock calendar card are jacked up.

![image](https://user-images.githubusercontent.com/61926834/106346489-31bf8080-627d-11eb-85f2-bdcfdcdf56c2.png)
![image](https://user-images.githubusercontent.com/61926834/106346500-456ae700-627d-11eb-89d4-60baa67284f5.png)
![image](https://user-images.githubusercontent.com/61926834/106346503-50257c00-627d-11eb-9e8f-1e5f3c1fee60.png)

With Atomic Calendar Revival from HACS, looks great!  I recommend folks just use that, it's a better card overall.  (I have an issue with UTC on all day events currently and I stripped a lot of options out of my view from how it comes by default but this is still a decent example)

![image](https://user-images.githubusercontent.com/61926834/106346550-97ac0800-627d-11eb-9b5e-cb533bea9f17.png)

**Comment by thatkookooguy** [comment]
@F1DNA yeah, that's the one I use as well... that's why I didn't notice this issue.

But I still can't get the same white bars you get in your screenshots

**Comment by thatkookooguy** [comment]
I guess recommending the other calendar, for now, is a good temporary solution. I'll keep trying to investigate this, but it might be more challenging to fix than just changing a color.
